src/Projection/PdoEventStoreProjector.php

```diff
@@ -38,6 +38,10 @@
 
 final class PdoEventStoreProjector implements Projector
 {
+    public const OPTION_PCNTL_DISPATCH = 'trigger_pcntl_dispatch';
```

Comment:
Thought about that too, they definitely belong there and are helpful for others. This would require a new release of the event-store and update the minimum version required in this package though (else the constants in the projection managers are not defined). If you prefer this I certainly can draft this up.